app/src/ActionModal/components/ActionModal.vue

```diff
@@ -693,6 +693,7 @@ export default {
           }
           polkadotAPI = await getPolkadotAPI(this.network)
         }
+        const accountType = await this.$store.dispatch(`getCurrentAccountType`)
```

Comment:
do this on signing in to that address. here it should already be set in the store